Tests/AppTests/AllTests.swift

```diff
@@ -15,12 +15,14 @@
 @testable import App
 
 import Dependencies
+import SnapshotTesting
 import Testing
 
 
 @Suite(
     .dependency(\.date.now, .t0),
-    .dependency(\.metricsSystem, .mock)
+    .dependency(\.metricsSystem, .mock),
+    .snapshots(record: .failed)
```

Comment:
This is a great quality of life change: The if a snapshot differs, it will automatically record a new snapshot and then _fail_.

That means that in order to update snapshots, we don't need to edit tests anymore. We just need to check if there are snapshot differences and commit or revert them as needed.